apps/src/applab/mode-javascript_codeorg.js

```diff
@@ -0,0 +1,38 @@
+ace.define("ace/mode/javascript_codeorg",["require","exports","module","ace/lib/oop","ace/mode/javascript","ace/mode/javascript_highlight_rules","ace/worker/worker_client","ace/mode/matching_brace_outdent","ace/mode/behaviour/cstyle","ace/mode/folding/cstyle","ace/config","ace/lib/net"], function(acerequire, exports, module) {
+
+var oop = acerequire("ace/lib/oop");
```

Comment:
use acerequire instead of require so that browserify doesn't try to load these modules. If we were to add acerequires for ace modules that aren't in mode-javascript.js, those will fail at runtime.